_locales/pl/messages.json

```
@@ -0,0 +1,387 @@
+	},
+	"explore_exploring": {
+		"message": "Odkrywanie kolejki… Zapytanie $1 z $2"
+	},
+	"explore_failed_to_clear": {
+		"message": "Nie udało się wyczyścić elementu kolejki #$1, ponowna próba za pięć sekund…"
+	},
+	"explore_failed_to_clear_too_many": {
+		"message": "Nie udało się wyczyścić elementu kolejki #$1 zbyt wiele razy. Możesz spróbować ponownie później."
+	},
+	"explore_failed_to_generate": {
+		"message": "Nie udało się wygenerować nowej kolejki, ponowna próba za pięć sekund…"
+	},
+	"explore_finished": {
+		"message": "Kolejka została odkryta."
+	},
+
+	"badges_idle_apps": {
+		"message": "Aplikacje do idlowania: $1"
+	},
+	"badges_idle_apps_on_this_page": {
+		"message": "Aplikacje do idlowania na tej stronie: $1"
+	},
+	"badges_idle_drops": {
+		"message": "Pozostałe dropy: $1"
+	},
+	"badges_idle_drops_on_this_page": {
+		"message": "Pozostałe dropy na tej stronie: $1"
+	},
+
+	"achievement_your_progress": {
+		"message": "Twój postęp: $1 / $2"
+	},
+	"hidden_achievement": {
+		"message": "Ukryte osiągnięcie: "
+	},
+	"hidden_achievement_hover": {
+		"message": "Ukryte osiągnięcie, najedź myszką aby je ujawnić: "
+	},
+
+	"inventory_list_at": {
+		"message": "Wystaw za %price%"
+	},
+	"inventory_sell_at": {
+		"message": "Sprzedaj za %price%"
+	},
+	"inventory_list_at_title": {
+		"message": "Wystawia przedmiot po najniższej, podanej cenie sprzedaży\n\nWyświetlona cena to kwota, którą otrzymasz (bez opłat)"
+	},
+	"inventory_sell_at_title": {
+		"message": "Wystawia przedmiot za najwyższą podaną cenę zlecenia kupna\n\nWyświetlana cena to kwota, którą otrzymasz (bez opłat)."
+	},
+
+	"widget_view_app": {
+		"message": "Wyświetl aplikacje $1 na SteamDB"
+	},
+	"widget_view_sub": {
+		"message": "Wyświetl paczkę $1 on SteamDB"
+	},
+
+
+	"options": {
+		"message": "Opcje"
+	},
+	"options_welcome_title": {
+		"message": "👋 Dzięki za zainstalowanie rozszerzenia SteamDB!"
+	},
+	"options_welcome_description": {
+		"message": "Przejrzyj poniższe opcje i przełącz te, które preferujesz. Aby wrócić tutaj w przyszłości:"
+	},
+	"options_welcome_option1": {
+		"message": "Strona ustawień na SteamDB"
+	},
+	"options_welcome_option2": {
+		"message": "Rozwijane menu konta na stronach Steam"
+	},
+	"options_welcome_option3": {
+		"message": "Strona rozszerzeń twojej przeglądarki"
+	},
+	"options_review": {
+		"message": "Wystaw recenzje dla tego rozszerzenia"
+	},
+	"options_permissions_description": {
+		"message": "To rozszerzenie wymaga dodatkowych uprawnień do poprawnego działania w SteamDB, Sklepie Steam i Społeczności Steam."
+	},
+	"options_permissions_button": {
+		"message": "⚠️ Przyznaj uprawnienia"
+	},
+
+	"options_footer1": {
+		"message": "Stworzone z 💙 przez zespół <a href=\"https://steamdb.info\" target=\"_blank\">SteamDB</a>"
+	},
+	"options_footer2": {
+		"message": "To rozszerzenie jest <a href=\"https://github.com/SteamDatabase/BrowserExtension\" target=\"_blank\">open source</a>, dziękuję wszystkim kontrybutorom!"

```

```suggestion
		"message": "To rozszerzenie jest <a href=\"https://github.com/SteamDatabase/BrowserExtension\" target=\"_blank\">otwarto-źrodłowe</a>, dziękuję wszystkim zaangażowanym!"

```

Kontrybutor to rzadko spotykane słowo i dziwnie brzmi w tym kontekście, lepiej odnieść się do społeczności.